src/TableHeader.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+const tableTitle = [
```

Comment:
I'd call this variable something like `tableColumns` - but its great that you've abstracted this out into a separate variable. Well done